src/ApiTestCase.php

```diff
@@ -96,6 +108,8 @@ public function setUp()
     {
         parent::setUp();
         $this->fixtureLoader = new Loader();
+        $this->responseContentMatcher = new ResponseContentMatcher(new DefaultMatcher());
```

Comment:
Won't it be better to extract it to separate set up method? `setUpMatchet` for instance? This way we won't be forced to copy-paste whole setUp, and instead we will be able to override only direct setUp. WDYT?